core/connection_result.h

```diff
@@ -1,5 +1,9 @@
 #pragma once
 
+
+/**
+ * @brief Namespace for all dronecore types.
```

Comment:
@JonasVautherin FYI, namespace documentation is mandatory for doxygen to be able to generate the contained global objects.